src/dmd/backend/cdef.d

```diff
@@ -104,20 +104,24 @@ enum IMPLIED_PRAGMA_ONCE = 1;       // include guards count as #pragma once
 enum bool HEADER_LIST = true;
 
 // Support generating code for 16 bit memory models
-version (SCPP)
-    enum SIXTEENBIT = TARGET_WINDOS != 0;
-else
-    enum SIXTEENBIT = false;
-
+bool SIXTEENBIT()
+{
+    version (SCPP)
+        return TARGET_WINDOS != 0;
+    else
+        return false;
+}
 /* Set for supporting the FLAT memory model.
  * This is not quite the same as !SIXTEENBIT, as one could
  * have near/far with 32 bit code.
  */
-version (MARS)
-    enum TARGET_SEGMENTED = false;
-else
-    enum TARGET_SEGMENTED = TARGET_WINDOS;
-
+bool TARGET_SEGMENTED()
```

Comment:
and this

Reply:
I'm not sure about this. Please leave the SIXTEENBIT changes to me, as I'll ensure the front end keeps working.

Reply:
done